README.md

```diff
@@ -2192,8 +2180,6 @@ Bookmarks
 
 [Linux Device Drivers, 3rd Edition: Jonathan Corbet, Alessandro Rubini, Greg Kroah-Hartman: 9780596005900: Amazon.com: Books](https://www.amazon.com/Linux-Device-Drivers-Jonathan-Corbet/dp/0596005903/ref=sr_1_4?ie=UTF8&qid=1483650712&sr=8-4&keywords=understanding+linux+kernel)
 
-[Linux Kernel Teaching — The Linux Kernel documentation](https://linux-kernel-labs.github.io/master/)
```

Comment:
The link has been updated to this: https://linux-kernel-labs.github.io/refs/heads/master/